machines/vcsa/create-cluster.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash -e
+#
+# Copyright 2016 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Configure a vCenter cluster with vSAN datastore, DVS and DVPGs
+
+export GOVC_INSECURE=1
+export GOVC_USERNAME=${GOVC_USERNAME:-"Administrator@vsphere.local"}
+if [ -z "$GOVC_PASSWORD" ] ; then
+    # extract password from $GOVC_URL
```

Comment:
Too much parsing of GOVC_URL here and other scripts.  Made this PR to simplify: https://github.com/vmware/govmomi/pull/497

Just need to figure out why Travis and Drone are unhappy.